### PR TITLE
Glossary updates

### PR DESCRIPTION
**Pull request by AndreaFilyo**
Added new terms related to Inventory API and payment SDKs.

Related to Jira ticket https://izettle.atlassian.net/browse/DEVX-745

Note: Did not care about the italic formatting for now (feedback from the accessibility tests), since we will move everything to the Dev Portal going forward. The output will most likely be different there.

### CODE REVIEW

File: glossary.md [hunk 1]
```diff
@@ -9,10 +9,26 @@ Zettle Go API Glossary
...
+
+<dl>
+  <dt>Extra amount tipping flow</dt>
+  <dd>This flow prompts customers with a question before entering the tip amount. This tipping flow can be used when taking card payments using the Zettle SDK. See Total amount tipping flow.</dd>
 </dl>
 
 ## I
+<dl>
+  <dt>Inventory balance</dt>
+  <dd>The sum of all product items available at a specific location. The inventory balance for the store location type is usually the most important for integrations.</dd>
```
Comment (Astro-Choco) [inline glossary.md:29]: Shall we add "see Location"?

File: glossary.md [hunk 2]
```diff
@@ -24,28 +40,90 @@ Zettle Go API Glossary
...
 ## M
 <dl>
   <dt>Minor currency unit</dt>
   <dd>Also known as minor unit. A minor currency unit is the smallest unit of a currency. For example, for two-decimal currencies like US dollars, 199 with currency USD means $1.99 at Zettle. For zero-decimal currencies like Japanese Yen, 1095 with currency JPY means ¥1095 at Zettle.</dd>
 </dl>
 
 ## P
+<dl>
+  <dt>Partial return</dt>
+  <dd>Means that goods that is part of an original sale is being returned. In such a case, an amount is normally resolved and refunded from the original payment. See Refund.</dd>
```
Comment (Astro-Choco) [inline glossary.md:62]: General comment - it may be more readable with full sentences. Fpr example, A partial refund means ...

File: glossary.md [hunk 3]
```diff
@@ -24,28 +40,90 @@ Zettle Go API Glossary
...
+</dl>
+
 <dl>
   <dt>Preliminary account</dt>
   <dd>A Zettle merchant account. It's used to hold funds that are being checked by Zettle. After a transaction happens, the funds are recorded in the preliminary account while Zettle is checking whether the funds should be paid out. After that, the funds will be moved to and recorded in the liquid account while waiting to be paid out.</dd>
 </dl>
 
+<dl>
+  <dt>Purchase</dt>
+  <dd>The acquisition of an item or service which is typically paid for through an exchange of money or other asset.</dd>
```
Comment (Astro-Choco) [inline glossary.md:77]: which => that

**Review by Astro-Choco — APPROVED** [review]
Looks great :D

File: glossary.md [hunk 4]
```diff
@@ -9,10 +9,26 @@ Zettle Go API Glossary
...
 <dl>
   <dt>Acquiring bank</dt>
-  <dd>Also known as acquirer. An acquiring bank is a financial institution that authenticates the card holder and authorises the transaction. It makes sure that the card is valid and the card has sufficient funds to complete the transaction.</dd>
+  <dd>Also known as acquirer. An acquiring bank is a financial institution that authenticates the card holder and authorises the transaction. It ensures that the card is valid and the card has sufficient funds to complete the transaction.</dd>
+</dl>
+
+## E
+<dl>
+  <dt>Exchange</dt>
+  <dd>Instead of refunding payment as result of buyer returning goods from original sale the buyer and seller might agree on buyer receiving some other goods. This is called an exchange.</dd>
```
[the remark this answers is not in the document]
Comment (AndreaFilyo) [reply glossary.md:18]: Fixed, rephrased the whole thing

File: glossary.md (hunk 1 above)
Comment (AndreaFilyo) [reply glossary.md:29]: Fixed

File: glossary.md (hunk 2 above)
Comment (AndreaFilyo) [reply glossary.md:62]: Fixed

File: glossary.md (hunk 3 above)
Comment (AndreaFilyo) [reply glossary.md:77]: Fixed

**Review by Astro-Choco — APPROVED** [review]
Yay! Finally more glossary entries :D

**Review by dmilovanovi — APPROVED** [review]
Only thing I noticed is that we use phrases like seller/merchant & buyer/customer when referring to our personas. In my opinion that could be a bit confusing at times.
Everything else looks good to me 👍

File: glossary.md [hunk 5]
```diff
@@ -24,28 +40,90 @@ Zettle Go API Glossary
...
 ## M
 <dl>
   <dt>Minor currency unit</dt>
   <dd>Also known as minor unit. A minor currency unit is the smallest unit of a currency. For example, for two-decimal currencies like US dollars, 199 with currency USD means $1.99 at Zettle. For zero-decimal currencies like Japanese Yen, 1095 with currency JPY means ¥1095 at Zettle.</dd>
 </dl>
 
 ## P
+<dl>
+  <dt>Partial return</dt>
+  <dd>A partial return means that goods that is part of an original sale is being returned. In such a case, an amount is normally resolved and refunded from the original payment. See Refund.</dd>
```
[the remark this answers is not in the document]
Comment (dmilovanovi) [reply glossary.md:62]: Not sure if I'm right, but my eyes spotted this and it got me thinking 🤔

[the remark this answers is not in the document]
Comment (AndreaFilyo) [reply glossary.md]: You're absolutely correct...thank you

**Review by Astro-Choco — APPROVED** [review]
Looks great! :D

**Pull request merged**